Makefile

```diff
@@ -123,8 +122,42 @@ clean:
 	@echo "> Cleaning workspace..."
 	rm -rf $(BUILD_DIR)
 
-package: descriptor
-	@echo "> Packaging lifecycle..."
-	tar czf $(BUILD_DIR)/$(ARCHIVE_NAME).tgz -C $(BUILD_DIR) lifecycle.toml lifecycle
+package: package-linux package-windows
+
+package-linux: export LIFECYCLE_DESCRIPTOR:=$(LIFECYCLE_DESCRIPTOR)
+package-linux: GOOS:=linux
+package-linux: GOOS_DIR:=$(BUILD_DIR)/$(GOOS)
+package-linux: ARCHIVE_NAME=lifecycle-v$(LIFECYCLE_VERSION)+$(GOOS).x86-64
+package-linux:
+	@echo "> Writing descriptor file for $(GOOS)..."
+	mkdir -p $(GOOS_DIR)
+	echo "$${LIFECYCLE_DESCRIPTOR}" > $(GOOS_DIR)/lifecycle.toml
+
+	@echo "> Packaging lifecycle for $(GOOS)..."
+	tar czf $(BUILD_DIR)/$(ARCHIVE_NAME).tgz -C $(GOOS_DIR) lifecycle.toml lifecycle
+
+package-windows: export LIFECYCLE_DESCRIPTOR:=$(LIFECYCLE_DESCRIPTOR)
+package-windows: GOOS:=windows
+package-windows: GOOS_DIR:=$(BUILD_DIR)/$(GOOS)
+package-windows: ARCHIVE_NAME=lifecycle-v$(LIFECYCLE_VERSION)+$(GOOS).x86-64
+package-windows:
+	@echo "> Writing descriptor file for $(GOOS)..."
+	mkdir -p $(GOOS_DIR)
+	echo "$${LIFECYCLE_DESCRIPTOR}" > $(GOOS_DIR)/lifecycle.toml
+
+	@echo "> Packaging lifecycle for $(GOOS)..."
+	tar czf $(BUILD_DIR)/$(ARCHIVE_NAME).tgz -C $(GOOS_DIR) lifecycle.toml lifecycle
+
+package-darwin: export LIFECYCLE_DESCRIPTOR:=$(LIFECYCLE_DESCRIPTOR)
```

Comment:
I can't think of any situations where we need/want a darwin package, but also it's harmless, so we can keep it for consistency.

Reply:
I forget exactly how we landed on adding it. I'll remove it as it's extremely easy to add back.